grouper/fe/template_util.py

```diff
@@ -11,22 +11,27 @@ def _make_date_obj(date_obj):
     """Given either a datetime object, float date/time in UTC unix epoch, or
     string date/time in the form '%Y-%m-%d %H:%M:%S.%f' return a datetime
     object."""
+    if date_obj == '' or date_obj is None:
```

Comment:
I think this check should happen somewhere else, probably `print_date`? Otherwise this method returns a datetime *or* a string.